src/data/h1/tags/unit.yml

```diff
@@ -261,7 +261,11 @@ comments:
         - name: dialogue
           en: ...
     - name: grenade velocity
-      en: ...
+      en: >
+        The velocity of thrown grenades. This field is used for the player, but
+        overidden by [actor_variant _grenade velocity_][actor_variant#tag-field-grenade-velocity]
+        for AI. The [projectile _initial velocity_][projectile#tag-field-initial-velocity]
+        for grenades is not used.
```

Comment:
Interesting tid-bit. The globals tag can actually reference non-projectile tags for the grenade projectile field if modified outside of guerilla. An example would be a vehicle. Only problem is that in multiplayer since the client expects it to be a grenade it causes a crash if someone drives the vehicle into its owner (the thrower).

Reply:
Interesting... I'll add this note when the globals page gets a pass of its own.